web/src/components/molecules/ProjectOverview/ModelDeletionModal/index.tsx

```diff
@@ -0,0 +1,41 @@
+import Alert from "@reearth-cms/components/atoms/Alert";
+import Button from "@reearth-cms/components/atoms/Button";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import { Model } from "@reearth-cms/components/molecules/ProjectOverview";
+import { useT } from "@reearth-cms/i18n";
+
+export interface Props {
+  open: boolean;
+  model?: Model;
+  onClose: (refetch?: boolean) => void;
+  onDelete: (modelId?: string) => Promise<void> | void;
+}
+
+const ModelDeletionModal: React.FC<Props> = ({ open, model, onClose, onDelete }) => {
+  const t = useT();
+
+  return (
+    <Modal
+      title={t("Delete Model")}
+      visible={open}
+      onCancel={() => onClose()}
+      footer={[
+        <Button key="back" onClick={() => onClose()}>
+          {t("Cancel")}
+        </Button>,
+        <Button key="submit" type="primary" onClick={() => onDelete(model?.id)} danger>
+          {t("Delete Model")}
+        </Button>,
+      ]}>
+      <p>{t("Are you sure you want to delete this model ") + model?.name + "?"}</p>
```

Comment:
The final should be:

Are you sure you want to delete the model **SomeModel123**?

You might need to wrap the model?.name in a span and change the styles for it to be bolded (like the design).

Reply:
@KaWaite , changed it to bold.

Reply:
A you missed a small change. Not `...this model Blahblah` but `...the model Blahblah`